README.md

```diff
@@ -274,6 +252,20 @@ will be thrown, and any error(s) returned by the SmarterU API will be listed in
 the exception message. A list of all possible SmarterU errors can be found
 [here](https://support.smarteru.com/docs/api-error-codes).
 
+## Contributing
+
+If you would like to make a contribution to this library, you may do so using
+the following process:
+
+1. Clone the project to your own machine.
```

Comment:
I would suggest they fork the project rather than cloning the repo. Because a random user is unlikely to have write permissions to the repo, they would be unable to upload their branch in order to create the PR.

https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow